src/LiveComponent/assets/src/Backend/RequestBuilder.ts

```diff
@@ -26,9 +27,15 @@ export default class {
             Accept: 'application/vnd.live-component+html',
         };
 
+        const totalFiles = Object.entries(files).reduce(
```

Comment:
Could just be?

`const totalFiles = Object.keys(files).length`

Reply:
I think it's not - that would count `FileList` objects passed, but they can be empty (like inputs with no files selected). In that case we should not count them.

Actually maybe what would be more fitting is to first filter `files` to remove empty `FileList`s and then just use what you propose.